sdk/core/Azure.Core/README.md

```diff
@@ -217,6 +217,16 @@ KeyVaultSecret secret = client.GetSecret("Name");
 
 More on mocking in [mocking samples](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Mocking.md)
 
+## Distributed tracing with ApplicationInsights
```

Comment:
```suggestion
## Distributed tracing with Application Insights
```

I'm not sure if it's a branding thing, but we're using `Application Insights"  on line 222 and elsewhere in docs...